src/applications/static-pages/download-1095b/tests/01-1095b-authed.cypress.spec.js

```diff
@@ -59,4 +59,18 @@ describe('Authed 1095-B Form Download PDF', () => {
 
     cy.axeCheck();
   });
+
+  it('displays an error message when the PDF download fails', () => {
```

Comment:
I think this error can also occur on page load if the available forms endpoint fails. Not sure if that's worth a separate test because they're more or less the same.